qlib/workflow/task/collect.py

```diff
@@ -109,6 +112,27 @@ def load(filepath):
             raise TypeError(f"The instance of {type(collector)} is not a valid `Collector`!")
 
 
+class HyperCollector(Collector):
+    """
+    A collector to collect the results of other Collectors
+    """
+
+    def __init__(self, collector_dict, process_list=[]):
+        """
+        Args:
+            collector_dict (dict): the dict like {collector_key, Collector}
+            process_list (list or Callable): the list of processors or the instance of processor to process dict.
+        """
+        super().__init__(process_list=process_list)
+        self.collector_dict = collector_dict
+
+    def collect(self) -> dict:
+        collect_dict = {}
+        for key, collector in self.collector_dict.items():
+            collect_dict[key] = collector()
+        return collect_dict
```

Comment:
Will the format of the return value align with the definition of the interface ?